log/logcontroller.go

```diff
@@ -1,308 +0,0 @@
-package log
-
-import (
-	"crypto/md5"
-	"encoding/json"
-	"github.com/opencord/voltha-lib-go/v2/pkg/log"
-	"os"
-	"strings"
-	"time"
-)
-
-type ComponentLogController struct {
-	ComponentName       string
-	componentNameConfig *ComponentConfig
-	globalConfig        *ComponentConfig
-	configManager       *ConfigManager
-	logHash             [16]byte
-	hashActive          bool
-}
-
-//create new component LogController
-func NewComponentLogController(cm *ConfigManager) (*ComponentLogController, error) {
-	//populate the ComponentLogController with env variables
-	componentName := os.Getenv("COMPONENTNAME")
-	hash := [16]byte{}
-
-	cc := &ComponentLogController{
-		ComponentName:       componentName,
-		componentNameConfig: nil,
-		globalConfig:        nil,
-		configManager:       cm,
-		logHash:             hash,
-		hashActive:          false}
-
-	return cc, nil
-}
-
-//ProcessLogConfigChange initialize component config and global config then process the log config changes
-func ProcessLogConfigChange(cm *ConfigManager) {
-	//populate the ComponentLogController using NewComponentLogController
-
-	cc, err := NewComponentLogController(cm)
-	if err != nil {
-		log.Error("error", err)
-	}
-
-	ccGlobal, err := InitComponentConfig("global", ConfigTypeLogLevel, cm)
-	if err != nil {
-		log.Errorw("fail-to-create-component-config", log.Fields{"error": err})
```

Comment:
return error if this fails. add second err parameter as return value.

Also, error string should be fail-to-initialize-global-loglevel-config